definitions/npm/react-testing-library_v3.x.x/flow_v0.60.0-/test_react-testing-library_v3.x.x.js

```diff
@@ -0,0 +1,330 @@
+// @flow
+
+import React from 'react';
+import { render, Simulate, wait, fireEvent, renderIntoDocument, cleanup, waitForElement } from 'react-testing-library';
+import { describe, it } from 'flow-typed-test';
+import { domainToASCII } from 'url';
+
+describe('wait()', () => {
+  it('should fail on invalid inputs', () => {
+    // $ExpectError
+    wait(1);
+    // $ExpectError
+    wait(() => {}, 1);
+  });
+
+  it('should pass on correct inputs', () => {
+    wait(() => {});
+    wait(() => {}, { timeout: 1 });
+  });
+});
+
+describe('waitForElement()', () => {
+  it('should fail on invalid inputs', () => {
+    // $ExpectError
+    waitForElement(1);
+    // $ExpectError
+    waitForElement(() => {}, 1);
+  });
+
+  it('should pass on correct inputs', () => {
+    waitForElement(() => document.createElement('div'));
+    waitForElement(() => document.createElement('div'), {
+      container: document.createElement('div'),
+      timeout: 100
+    });
+  });
+});
+
+describe('render()', () => {
```

Comment:
A small nit, but generally I've seen it as `#render` or `render` but not `render()`. Change it if you want.

Reply:
@AndrewSouthpaw done :heavy_check_mark: